src/Nri/Ui/Carousel/V1.elm

```diff
@@ -0,0 +1,94 @@
+module Nri.Ui.Carousel.V1 exposing (..)
+
+import Css exposing (..)
+import Html.Styled as Html exposing (Html)
+import Nri.Ui
+import Nri.Ui.Tooltip.V3 as Tooltip
+import TabsInternal.V2 as TabsInternal
+
+
+type Tab id msg
+    = Tab (TabsInternal.Tab id msg)
+
+
+type Attribute id msg
+    = Attribute (TabsInternal.Tab id msg -> TabsInternal.Tab id msg)
+
+
+type TabPosition
+    = Before
+    | After
+
+
+{-| Tooltip defaults: `[Tooltip.smallPadding, Tooltip.onBottom, Tooltip.fitToContent]`
+-}
+withTooltip : List (Tooltip.Attribute msg) -> Attribute id msg
+withTooltip attributes =
+    Attribute (\tab -> { tab | tabTooltip = attributes })
+
+
+{-| Makes it so that the tab can't be clicked or focused via keyboard navigation
+-}
+disabled : Bool -> Attribute id msg
+disabled isDisabled =
+    Attribute (\tab -> { tab | disabled = isDisabled })
+
+
+{-| -}
+slideHtml : Html msg -> Attribute id msg
+slideHtml content =
+    Attribute (\tab -> { tab | panelView = content })
+
+
+labelledBy : String -> Attribute id msg
+labelledBy labelledById =
+    Attribute (\tab -> { tab | labelledBy = Just labelledById })
+
+
+tabHtml : Html Never -> Attribute id msg
+tabHtml content =
+    Attribute (\tab -> { tab | tabView = [ Html.map never content ] })
+
+
+{-| -}
+buildTab : { id : id, idString : String } -> List (Attribute id msg) -> Tab id msg
+buildTab config attributes =
+    Tab (TabsInternal.fromList config (List.map (\(Attribute f) -> f) attributes))
+
+
+view :
+    { focusAndSelect : { select : id, focus : Maybe String } -> msg
+    , selected : id
+    , tabs : List (Tab id msg)
+    , tabStyles : Int -> Bool -> List Style
```

Comment:
From an API perspective, I think we should consider adjusting `tabStyles` a bit.

1. It's a little bit surprising that the styles can change based on the _index_ of the carousel control (Although I'm not sure it's obvious that this is what is happening). I don't think it's typical for carousel controls to have differing styles based on index. 
2. Other details about the carousel control are managed by the Attributes pattern. I'd expect for the to follow either the config-heavy approach completely or to follow the Attributes-style approach completely.

This is the pattern I mean by `Attributes-style`:
```
css : List Style -> Attribute id msg
```

Reply:
1. Agreed, I'll change it to be `Bool -> List Style`
2. I understand and agree, and would like to follow an Attributes-style approach, but, at the same time, I don't think there's a simple way to use in this case, as the `tabStyles` are passed to `TabsInternal.view`, which is responsible for rendering the actual controls. In any case, I would like to put some effort into this in the near future, but we are kind of in a hurry with the marketing website 😅

Reply:
> What do you think about "control" for the tab and "slide" for the tab panel?

Sounds good!

> wondering if it makes sense to go config-heavy

Up to you! I like that the Attributes pattern makes extending components super simple and makes adding configuration options to the examples relatively simple. However, I'm not expecting Carousel to be used in so many places that optimizing for these nice-to-haves is necessary.

The config pattern definitely results in easier-to-follow internal code.

---

One thing I'm wondering: how configurable does the component actually need to be? Would it make sense to match the styles from the spec inside the Carousel component, rather than on the about page later?

Reply:
I'd prefer to implement the designs in the homepage instead, for two reasons:

* having it on the homepage allows us to iterate quicker on design feedback/changes
* the requirements for an accessible Carousel/Tab can be quite complicated to implement (specially in Elm). I believe having a configurable, yet simple, Carousel component can be of great help when we need one.